index.js

```diff
@@ -23,7 +23,6 @@ module.exports = {
   transaction: transaction.default,
   version: version.default,
   wallet: wallet.default,
-  WebSocketHandler: WebSocketHandler.default,
```

Comment:
Shouldn't we leave it here for compatibility?

Reply:
Yes, makes sense. Done